acceptance/internal/config.go

```diff
@@ -79,6 +79,12 @@ type TestConfig struct {
 
 	CompiledIgnoreObject *ignore.GitIgnore
 
+	// Environment variables
+	// If the same variable is defined both in Env and EnvMatrix, the one in EnvMatrix takes precedence
+	// regardless of which config file it is defined in.
+	// Note, keys are sorted alphabetically rather in order they are defined. This matter when one key reference another (same for EnvMatrix).
```

Comment:
nit: instead of commenting "(same for EnvMatrix)" here, it is better to add the same statement to the `EnvMatrix` doc comment itself